root/documentation/info.conf

```diff
@@ -165,7 +165,57 @@
       </one>
     </examples>
   </biotypes>
-  
+
+  <biotypes_groups>
+    description=List the functional classifications within a biotype group. Without argument the list of available biotype groups is returned.
+    endpoint="info/biotypes/groups/:group"
+    method=GET
+    group=Information
+    output=json
+    <params>
+      <group>
+        type=String
+        description=Biotype group
+        required=0
+        example=__VAR(biotype_group)__
+      </group>
+    </params>
+    <examples>
+      <one>
+        path=/info/biotypes/groups/
+        content=application/json
+      </one>
+      <two>
+        path=/info/biotypes/groups/
+        capture=__VAR(biotype_group)__
+        content=application/json
+      </two>
+    </examples>
+  </biotypes_groups>
+
+  <biotypes_name>
+    description=List the biotypes for a given biotype name. Usually two exist for a particular name, based on the object_type (gene or transcript). 
```

Comment:
I would like the wording here to be changed to something like "List the properties of biotypes with a given name".